### PR TITLE
Tooltip: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

**Comment by ksercs** [comment]
There is "_devextreme-documentation\concepts\05 Widgets\Tooltip\10 Customize the Content\Customize the Content.md_" empty file. Should we remove it?

**Comment by ksercs** [comment]
Tooltip target image is unavailable now. Should we change the image source to real and add it to the markup in Vue and React code snippets?

File: concepts/05 Widgets/Tooltip/10 Customize the Content/5 Switching Templates On the Fly.md [hunk 1]
```diff
@@ -81,6 +81,112 @@ If you need to render different templates depending on a specific condition, def
...
+            DxButton
+        },
+        data() {
+            return {
+                currentTemplate: "template1"
+            };
+        },
+        methods: {
+            changeTemplate () {
+                this.currentTemplate = (this.currentTemplate === 'template1' ? 'template2' : 'template1')
```
Comment (AlexJameson) [inline concepts/05 Widgets/Tooltip/10 Customize the Content/5 Switching Templates On the Fly.md]:
```suggestion
                this.currentTemplate = (this.currentTemplate === 'template1' ? 'template2' : 'template1');
```

**Review by AlexJameson — APPROVED** [review]

**Comment by AlexJameson** [comment]
> 
> 
> Tooltip target image is unavailable now. Should we change the image source to real and add it to the markup in Vue and React code snippets?

I don't think so.

**Comment by AlexJameson** [comment]
> 
> 
> There is "_devextreme-documentation\concepts\05 Widgets\Tooltip\10 Customize the Content\Customize the Content.md_" empty file. Should we remove it?

We should keep it for some menu rendering related issues.

File: concepts/05 Widgets/Tooltip/10 Customize the Content/5 Switching Templates On the Fly.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/Tooltip/10 Customize the Content/5 Switching Templates On the Fly.md]: fixed

File: concepts/05 Widgets/Tooltip/00 Overview.md [hunk 2]
```diff
@@ -51,6 +51,67 @@ The following code creates a simple **Tooltip** on your page and attaches it to
         // ...
     })
 
+##### Vue
+
+    <template>
+        <div>
+            <img id="image" src="https://www.devexpress.com/DXR.axd?r=9999_17-FD0Id" />
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Tooltip/00 Overview.md]:
```suggestion
            <img id="image" src="https://url/to/an/image" />
```

File: concepts/05 Widgets/Tooltip/05 Show and Hide the Tooltip/05 API.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ The **show()** method called without arguments shows the **Tooltip** for the [ta
 
 ---
 
-With Angular, AngularJS, or Knockout, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxTooltip/Configuration/#visible') property of the **Tooltip** widget to a component property (in Angular), a scope property (in AngularJS) or an observable variable (in Knockout). After that, change them, and the **Tooltip** will appear or disappear.
+With Angular, Vue, or React, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxTooltip/Configuration/#visible') property of the **Tooltip** widget to a component property. After that, change them, and the **Tooltip** will appear or disappear.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Tooltip/05 Show and Hide the Tooltip/05 API.md]:
```suggestion
With Angular, Vue, or React, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxTooltip/Configuration/#visible') property of the **Tooltip** widget to a component property. After that, change this component property, and the **Tooltip** will appear or disappear.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Tooltip/05 Show and Hide the Tooltip/05 API.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/Tooltip/05 Show and Hide the Tooltip/05 API.md]: fixed

File: concepts/05 Widgets/Tooltip/00 Overview.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/Tooltip/00 Overview.md]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**